docs/index.md

```diff
@@ -0,0 +1,5 @@
+# MyST-NB Quickstart
+
+```{toctree}
+Hello
```

Comment:
Can you rename `Hello.ipynb` to `hello.ipynb`? Now that the notebooks are integrated into a website, it is cleaner to make all notebook names lower-case and with dashes, as their paths reflect the eventual webpage URLs.

Reply:
Nice tips!